src/renderer/views/About/About.scss

```
@@ -62,3 +57,30 @@
     margin: 0;
   }
 }
+
+.logo {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  inline-size: 100%;
+
+  .logoIcon {
+    background-image: var(--logo-icon);
+    background-position: right top;
+    background-repeat: no-repeat;
+    background-size: 200px;
+    block-size: 200px;
+    inline-size: 200px;
+  }
+
+  .logoText {
+    background-image: var(--logo-text);
+    background-position: right top;
+    background-repeat: no-repeat;
+    background-size: 350px;
+    block-size: 175px;
+    position: relative;
+    inline-size: 350px;
+  }
+}
```

We don't need nesting for any of these classes, as Vue's scoped CSS feature (activated by the `scoped` attribute on the `<style>` tag) already takes care of making sure it only applies to this component.

Yes, I am aware that we have a bunch of unnecessary nesting in other files in the project, but I do eventually want to get rid of it, as it increases the size of the CSS output without providing any benefit.

```suggestion
}

.logoIcon {
  background-image: var(--logo-icon);
  background-position: right top;
  background-repeat: no-repeat;
  background-size: 200px;
  block-size: 200px;
  inline-size: 200px;
}

.logoText {
  background-image: var(--logo-text);
  background-position: right top;
  background-repeat: no-repeat;
  background-size: 350px;
  block-size: 175px;
  position: relative;
  inline-size: 350px;
}
```

---

updated to remove nesting